hack/prom-pushgateway/up.sh

```diff
@@ -34,7 +34,7 @@ if [ ! -f "${kube_prom_zip}" ]; then
 fi
 
 # unpack the operator
-tar -zxvf "${kube_prom_zip}" -C "${script_dir}"
+unzip "${kube_prom_zip}" -d "${script_dir}"
```

Comment:
Turns out on a Mac, `tar` symlinks to `bsdtar` not GNU tar. There are some minor differences there, so we're definitely not using the same tar.

`unzip` is a good replacement. 👍